src/value.rs

```diff
@@ -102,6 +102,22 @@ pub fn lua_to_rbxvalue(ty: VariantType, value: LuaValue<'_>) -> LuaResult<Varian
     }
 }
 
+pub fn guess_type_from_rbxvalue(value: &LuaValue<'_>) -> Option<VariantType> {
```

Comment:
I'm not ecstatic about this. I think it might be better to just make something like `setRawAttribute` in its place.

Reply:
Small note, but `RbxValue` is a very dead name now, we just call those variants now!